### ISSUE TITLE
2.12.0-beta1 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3888]
https://github.com/orgs/az-digital/projects/126

- #3843 
- #3840 
- #3853 
- #3815 
- #3875 
- #3864
- #3878 
- #3872 
- #3858
- #3867 
- #3871 

## Pick list
```
git cherry-pick c28506abb2d998e4170e2062b7bf17bded803f68
git cherry-pick 1eccee414c40c3ffb18702449d24a7db78cf8bb0
git cherry-pick 82d999d8b3f6c00990b4380e85341db0a32071de
git cherry-pick 906fe145924fd5b5271b80016b2f101e6fdeb54a
git cherry-pick d487d7124f7dc5c11450f3f8cc07ad1f5f3219e2
git cherry-pick 88cf30841de78fff9f3094b699892b480c4cde21
git cherry-pick fb9fdd8f41c28727efa6b887d0c450fa3a284800
git cherry-pick c1e5508b41efde162aa47e3b97423e141037a012
git cherry-pick ea1f77811b43d6f7434e33cdc7d928ba8b541643
git cherry-pick a5e2043d2e4b351bcc91b1f371753965279c749a
git cherry-pick 78e69a0815ddb9184d9389d8a425abc1c521faef
```

**Comment by trackleft** [comment]
LGTM

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.12.0-alpha1..HEAD --pretty='format:- %s'`
```
- Closes #3863 Update Global Footer Links (#3871)
- Close #3861 Change scope of spaceless filter from all elements to AZ Person headings (#3867)
- Update drupal/block_class requirement from 2.0.11 to 4.0.0 (#3858)
- Closes #3870 Revert #3219: Add anchor link plugin for CKEditor5 (#3872)
- Update drupal/smart_date requirement from 4.1.6 to 4.2.1 (#3878)
- Fixes #3823 Card Widget Deprecation Notices (#3864)
- Fixes #3874 Modules using Configuration Overrides should declare az_core dependency (#3875)
- Close #3813 set default format to plain_text in flexible page summary field migration. (#3815)
- Fixes #3852 Citation Style Group Rendering (#3853)
- Close #1479 exclude Drupal 6 migrations from appearing when running drush migrate:status (#3840)
- Close #3842 Content editors can now upload and use webp images with their content. (#3843)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.12.0-beta1
